app/lib/sharezone_plus/page/sharezone_plus_page_controller.dart

```diff
@@ -33,50 +35,38 @@ class SharezonePlusPageController extends ChangeNotifier {
     _purchaseService = purchaseService;
     _subscriptionService = subscriptionService;
 
-    _listenToPlusStatus();
-    _getPlusPrice();
+    // Fake loading for development purposes.
+    Future.delayed(const Duration(seconds: 1)).then((value) {
+      hasPlus = true;
+      price = fallbackPlusPrice;
+      notifyListeners();
+    });
   }
 
   /// Whether the user has a Sharezone Plus subscription.
   ///
-  /// We use `false` as the initial value because we don't know if the user has
-  /// a subscription or not. The value will be updated as soon as the
-  /// subscription status is fetched from the backend.
-  bool hasPlus = false;
-
-  /// The price of the Sharezone Plus subscription including the currency
-  /// symbol.
-  String price = fallbackPlusPrice;
-
-  Future<void> _getPlusPrice() async {
-    final product = await _purchaseService.getPlusSubscriptionProduct();
-
-    if (product != null) {
-      price = product.priceString;
-      notifyListeners();
-    }
-  }
-
-  void _listenToPlusStatus() {
-    final hasPlus = _subscriptionService.isSubscriptionActiveStream();
+  /// If `null` then the status is still loading.
+  bool? hasPlus;
 
-    _hasPlusSubscription = hasPlus.listen((hasPlus) {
-      this.hasPlus = hasPlus;
-      notifyListeners();
-    });
-  }
+  /// The price for the Sharezone Plus per month, including the currency sign.
+  ///
+  /// If the user is subscribed to Sharezone Plus then this is the price for
+  /// his current subscription.
```

Comment:
FYI: It's likely that we are not going to support this for the beginning because we don't know which subscription the user has when the user is on a different platform, e.g. on the web.

Reply:
Ok, it's easier than I thought to implement because RevenueCat includes this information into the Firestore document. But still something we haven't implemented on the backend site and it's not a blocker to launch from my side so probably we ship later as an update.

Reply:
Let's keep it for now, if it takes more than an hour to implement we can still remove it later.